doonut-app-external-api/src/main/kotlin/com/doonutmate/oauth/controller/OauthController.kt

```diff
@@ -23,4 +26,13 @@ class OauthController(
     fun login(@RequestParam oauthType: OauthType, @RequestBody loginRequest: LoginRequest): LoginResponse {
         return oauthService.login(loginRequest, oauthType)
     }
+
+    @GetMapping("login/1")
```

Comment:
그리고 이건 취향문제일 수도 있는데 url 앞에도 `/`을 붙여주실 수 있을까용

```suggestion
    @GetMapping("/login/1")
    }
```